docs/src/functions_overview.md

```diff
@@ -71,3 +72,9 @@ Here the *Special Functions* are listed according to the structure of [NIST Digi
 | [`logbeta(x,y)`](@ref SpecialFunctions.logbeta)                                          | accurate `log(beta(x,y))` for large `x` or `y`      |
 | [`logabsbeta(x,y)`](@ref SpecialFunctions.logabsbeta)                                          | accurate `log(abs(beta(x,y)))` for large `x` or `y`     |
 | [`logabsbinomial(x,y)`](@ref SpecialFunctions.logabsbinomial)                                          | accurate `log(abs(beta(x,y)))` for large `x` or `y`     |
+
+## Riemann Zeta and Related Functions
```

Comment:
Same: "Related Functions"  shouldn't be capitalized.

Reply:
fixed